src/components/coin.ts

```diff
@@ -61,6 +61,11 @@ export const coinBonusMap = new Map<BonusType, Bonus>([
   ]
 ]);
 
+export interface UserCoinEntry {
```

Comment:
I can't ctrl f on phone but can u just lmk where this is used XD

Reply:
In the leaderboard :))